packages/docs-examples/clamped-text/clamped-text-overview/clamped-text-overview-example.ts

```diff
@@ -0,0 +1,36 @@
+import { ChangeDetectionStrategy, Component, signal } from '@angular/core';
+import { KbqClampedText } from '@koobiq/components/clamped-text';
+import { KbqToggleModule } from '@koobiq/components/toggle';
+
+/**
+ * @title Clamped-text overview
+ */
+@Component({
+    selector: 'clamped-text-overview-example',
+    standalone: true,
+    imports: [KbqClampedText, KbqToggleModule],
+    template: `
+        <div class="layout-margin-bottom-l">
+            <kbq-clamped-text [isCollapsed]="collapsed()" (isCollapsedChange)="collapsed.set($event)">
+                {{ text }}
+            </kbq-clamped-text>
+        </div>
+
+        <kbq-toggle [checked]="collapsed()" (change)="collapsed.set($event.checked)">Collapsed</kbq-toggle>
+    `,
+    styles: `
+        :host > div {
+            overflow: auto;
+            resize: horizontal;
+            max-width: 100%;
+            min-width: 150px;
+            padding: var(--kbq-size-xxs);
+        }
+    `,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ClampedTextOverviewExample {
+    protected readonly collapsed = signal(true);
+    protected readonly text =
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim  veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea  commodo consequat. Duis aute irure dolor in reprehenderit in voluptate  velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint  occaecat cupidatat non proident, sunt in culpa qui officia deserunt  mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur  adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.';
```

Comment:
рыбу можно из tabs взять
```suggestion
        'In a distributed denial-of-service attack (DDoS attack), the incoming traffic flooding the victim originates from many different sources. More sophisticated strategies are required to mitigate this type of attack; simply attempting to block a single source is insufficient as there are multiple sources. A DoS or DDoS attack is analogous to a group of people crowding the entry door of a shop, making it hard for legitimate customers to enter, thus disrupting trade and losing the business money. Criminal perpetrators of DoS attacks often target sites or services hosted on high-profile web servers such as banks or credit card payment gateways. Revenge and blackmail, as well as hacktivism, can motivate these attacks.';
```